libs/blocks/marketo/marketo.js

```diff
@@ -186,4 +186,6 @@ const init = (el, loadScriptFunc = loadScript) => {
     });
 };
 
-export default init;
+export default function init(el) {
+  createIntersectionObserver({ el, callback: loadMarketo });
```

Comment:
We should use intersection observer on only loadScript so there isn't undecorated content and we have the min-height from `marketo-form-wrapper`. 
I would say just move the create tags into init but we need to pass formData into loadMarketo, so a closure:
```suggestion
  const fragment = new DocumentFragment();
  //...
  el.replaceChildren(fragment);
    
  createIntersectionObserver({ el, callback: (el, entry) => {
    loadMarketo(el, formData);
  } });
```